### PR TITLE
Cleaning up our map files

### PR DESCRIPTION
**Pull request by Tk420634**
- Adds a legacy storage for old .dmms
- renames all our main map files, thusly putting sunset wastelands biggest mark on our code where it belongs.  In the trash.

## Pre-Merge Checklist
- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Review by TalkingCactus — APPROVED** [review]

File: _maps/pahrump-everything.json [hunk 1]
```diff
@@ -144,21 +143,6 @@
 			"Force Y": 3,
 			"Force Z": 2
 		},
```
Comment (DameonOwen) [inline _maps/pahrump-everything.json:145]: Remove comma

Comment (Tk420634) [reply _maps/pahrump-everything.json:145]: Remove comma

File: _maps/_basemap.dm [hunk 2]
```diff
@@ -4,15 +4,14 @@
 
 #ifndef LOWMEMORYMODE
 	#ifdef ALL_MAPS
-		#include "map_files/Pahrump-Sunset/Dungeons.dmm"
```
Comment (DameonOwen) [inline _maps/_basemap.dm:7]: So, like, remove `_maps/` from the beginning of all of these imports and it should work.

Comment (Tk420634) [reply _maps/_basemap.dm:7]: i shan't

File: _maps/_basemap.dm [hunk 3]
```diff
@@ -4,15 +4,14 @@
...
-		#include "map_files/Pahrump-Sunset/Pahrump-Sunset-Lower.dmm"
-		#include "map_files/Pahrump-Sunset/Pahrump-Sunset.dmm"
-		#include "map_files/Pahrump-Sunset/Pahrump-Sunset-Upper.dmm"
-		#include "map_files/Pahrump-Sunset/Pahrump-Sunset-Upper-2.dmm"
-		#include "map_files/Pahrump-Sunset/RockSprings.dmm"
-		#include "map_files/Pahrump-Sunset/RockSprings-Upper.dmm"
-		#include "map_files/Pahrump-Sunset/RedRiver.dmm"
-		#include "map_files/Pahrump-Sunset/Garland-City.dmm"
+		#include "_maps/map_files/coyote-bayou/Dungeons.dmm"
```
Comment (DameonOwen) [inline _maps/_basemap.dm]: You renamed the folder `coyote_bayou`.

File: _maps/pahrump-everything.json [hunk 4]
```diff
@@ -144,21 +143,6 @@
...
-			"No Parallax": true,
-			"Linkage" : "Cross",
-			"Force East": 1,
-			"Name": "Garland City",
-			"Force X": 3,
-			"Force Y": 2,
-			"Force Z": 1
-		}
-	],
+	]
```
Comment (DameonOwen) [inline _maps/pahrump-everything.json:146]: This still needs a comma.

File: _maps/pahrump-everything.json (hunk 1 above)
Comment (DameonOwen) [inline _maps/pahrump-everything.json:145]: This should not have a comma

File: _maps/pahrump-everything.json (hunk 4 above)
Comment (Tk420634) [reply _maps/pahrump-everything.json:146]: wuh

Comment (Tk420634) [reply _maps/pahrump-everything.json]: wuh

**Pull request merged**